web/profiles/joinup/joinup.behat.inc

```diff
@@ -194,4 +244,63 @@ class JoinupSubContext extends DrupalSubContextBase implements DrupalSubContextI
     }
   }
 
+  /**
+   * Navigates to the canonical page display of a node entity.
+   *
+   * @param string $bundle
+   *   The type of the node page.
+   * @param string $title
+   *   The title of the node.
+   *
+   * @When (I )go to the :bundle (content ) :title edit screen
+   * @When (I )visit the :bundle (content ) :title edit screen
+   */
+  public function visitCollection($bundle, $title) {
+    /** @var \Drupal\node\NodeInterface $node */
+    $node = $this->getNodeByTitle($title, $bundle);
+    $this->visitPath($node->url('edit-form'));
+  }
+
+  /**
+   * Checks if a content entity is published.
+   *
+   * @param string $bundle
+   *   The node bunlde.
+   * @param string $title
+   *   The node title.
+   *
+   * @throws \Exception
+   *   Throws an exception if the content is not published.
+   *
+   * @Then the :title :bundle (content )should be published
+   */
+  public function nodePublished($bundle, $title) {
+    /** @var \Drupal\node\NodeInterface $node */
+    $node = $this->getNodeByTitle($title, $bundle);
+    if (!$node->isPublished()) {
+      throw new \Exception("$title content should be published but is not.");
```

Comment:
I think it is actually correct because 'but' connects the two sentences and the subject remains (content) the same.